dist/stat_data.py

```diff
@@ -703,18 +701,18 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_walks_gave_up_no_targets', 'eviction walks gave up because they saw too many pages and found no candidates'),
     CacheStat('cache_eviction_walks_gave_up_ratio', 'eviction walks gave up because they saw too many pages and found too few candidates'),
     CacheStat('cache_eviction_walks_stopped', 'eviction walks gave up because they restarted their walk twice'),
+    CacheStat('cache_hs_read', 'history store table update reads'),
     CacheStat('cache_inmem_split', 'in-memory page splits'),
     CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
-    CacheStat('cache_lookaside_read', 'cache overflow table update reads'),
-    CacheStat('cache_page_instantiate_read_lookaside', 'pages read into cache requiring cache overflow entries.'),
-    CacheStat('cache_page_instantiate_read_lookaside_checkpoint', 'pages read into cache by checkpoint requiring cache overflow entries.'),
+    CacheStat('cache_page_instantiate_read_hs', 'pages read into cache requiring history store entries.'),
+    CacheStat('cache_page_instantiate_read_hs_checkpoint', 'pages read into cache by checkpoint requiring history store entries.'),
```

Comment:
Does this stat still makes sense ? Even if we read a page which has history store entries, how that will impact ?

Reply:
No. I removed the stats and its usage. The function that uses these stats is not removed, as that cleanup is taken care in another ticket.